packages/api-elements/lib/utils.js

```diff
@@ -32,11 +32,18 @@ function hasTypeAttribute(e, attribute) {
 }
 
 /**
- * Check if element has 'fixed' or 'fixedType' typeAttribute set
+ * Check if element has 'fixed' typeAttribute set
  * @param {element} e - element
  * @return {boolean}
  */
-const isFixed = e => hasTypeAttribute(e, 'fixed') || hasTypeAttribute(e, 'fixedType');
+const isFixed = e => hasTypeAttribute(e, 'fixed');
```

Comment:
Unrelated to PR but just something I noticed while reading this again, would be nice to make `hasTypeAttribute` curried and in opposite argument order. I can make a PR once these are merged

```
const isFixed = hasTypeAttribute('fixed');
```

Reply:
I can do that on the [other PR](https://github.com/apiaryio/api-elements.js/tree/patricksmms/improve-define-value-of)